src/FormGroup/FormGroup.jsx

```diff
@@ -50,9 +73,16 @@ export default function FormGroup(props) {
       )}
       id={props.id}
     >
+      {renderFormGroupChildren()}
+    </div>
+  );
+
+  const renderFormGroupChildren = () => (
```

Comment:
```suggestion
  const formGroupChildren = (
```

Since the functional component is executed on every render there's not really a need for this to be inside a function. It could just be assigned to a const. Or if it starts to get more complicated, broken out into a child component